jinahub/segmenters/VADSpeechSegmenter/manifest.yml

```diff
@@ -0,0 +1,5 @@
+manifest_version: 1
+name: VADSpeechSegmenter
+description: VADSpeechSegmenter segments speech audio using Silero VAD (voice activity detector)
+url: https://jina.ai
```

Comment:
`url` should be `https://github.com/jina-ai/executors/tree/main/jinahub/segmenters/VADSpeechSegmenter`

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/ff4ce6f7b6f413350770714f6b67ffecbd8b1067) for the change, thank you.